.github/workflows/release-deploy.yml

```diff
@@ -0,0 +1,65 @@
+name: release-deploy
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [ '3.10' ]
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: main
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Build package
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install wheel
+        cd main
+        echo "RELEASE_VERSION=$(cat pyfakefs/_version.py | cut -d "'" -f 2)" >> $GITHUB_ENV
+        python setup.py sdist bdist_wheel
+
+    - name: Publish package to PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        user: __token__
+        password: ${{ secrets.PYPI_API_TOKEN }}
+
+    - name: Create release docs
+      run: |
+        python -m pip install wheel
+        python -m pip install pytest
+        python -m pip install sphinx
+        cd main/docs
+        make html
+
+    - name: Checkout gh-pages
+      uses: actions/checkout@v2
+      with:
+        ref: gh-pages
+        path: doc
+
+    - name: Copy and commit release documentation
+      run: |
+        cp -r main/gh-pages/* doc/release
```

Comment:
Ahh interesting, the docs are published in gh-pages.

In the future I suggest to use readthedocs.io, which builds the docs automatically for releases and even PRs for preview. 👍

Reply:
Until now, I have always done it this way (including in `pytest-order`), but I will have a look at readthedocs in this case, thanks. Not for this release, though...

Reply:
The preview feature is indeed something I am missing in gh-pages, so I will definitely check this out!

Reply:
Actually, setting up ReadTheDocs is very easy, as it turned out - I will probably change this before the release. Maybe tomorrow if I get to it... The PR preview is too good a feature to miss, and as far as I understand, there is literally nothing to do in the release workflow (apart from removing the GH Pages stuff again). Thanks for the tip!

Reply:
Sure thing!

The preview is not enabled by default, you need to go into advanced settings and enable it:

![image](https://user-images.githubusercontent.com/1085180/194726494-592aca4f-a3bf-4342-90ef-e4ec1a5cec4d.png)

Reply:
Thanks, I noticed this. A bit odd that this cannot be configured in the configuration file...